packages/web-components/src/components/rux-clock/rux-clock.tsx

```diff
@@ -91,10 +91,15 @@ export class RuxClock {
     }
 
     /**
-     * Applies a smaller clock style.
+     * Applies a smaller clock style. (Deprecated, use `size="small"` instead)
      */
     @Prop({ reflect: true }) small: boolean = false
 
+    /**
+     * Sets the clock element size `small` or `large. The default styling is large.
+     */
+    @Prop({ reflect: true }) size?: 'small' | 'large'
```

Comment:
So.. the default was just 'not small' because it was boolean. I don't know whether to remove other options and just have `size = 'small'` be the only option or make a default size called 'default' or what. because for the ones I've changed there are essentially only two sizes. 'small' and .. 'not small'. I'm happy to name 'not small' something like Default instead. I went with large because its the opposite of small but like.. that could be opinionated and cause issues in the future if we ever do add a 'large' size.

Reply:
Input and button have a `size: 'small' | 'medium' | 'large' = 'medium'`. I feel like that'd make sense here too? Medium being whatever it is now, and the default. 
imo these shouldn't be props at all though, so I'm not super opinionated about what you name them.

Reply:
@markacianfrani you can have the final say here unless @FMorrison87 has any strong opinions. I'm leaning toward mirroring how input and button do it, although I'm like a 5/10 on that

Reply:
I think medium should be the default. but also 1/10. If I want the default size, I would just omit the size attr entirely rather than explicitly set it to size=medium/large. unless I needed it to change dynamically

Reply:
I am 100% fine with medium being the default and omitting large. I can change that up really quick.